Standards/CCSMB-14.md

```diff
@@ -0,0 +1,56 @@
+# *CCSMB 14:* Stereo television
+
+*Author: @notbronwyn*  
+*Version: v1.0*  
+*Last updated: 2024-03-30*  
+
+## Server:
+Videos should either be in BIMG but other formats do not break compliance as long as they are marked in "type" to avoid issues with clients and there is an open-source encoder and decoder
+  
+  an example of this would be [PBB](https://raw.githubusercontent.com/craftytv/pigutv/main/pbbconverter.py) which is supported in the example server
+
+Broadcasts in BIMG should only use the default CraftOS pallete
```

Comment:
Why is this limit in place? If there's a good reason, sure, but this seems like an unneeded limitation

Reply:
Issues with 20fps video mainly
it works if you dont set the pallete though